com.maxint.maxint.yml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: com.maxint.maxint
+runtime: org.gnome.Platform
```

Comment:
Do you really need org.gnome.Platform? Can this work with freedesktop runtime?

Reply:
I tried using freedesktop. But it shows an error for the Flutter Embedder. It shows it can't find a glib symbol (I can't remember the name. Sorry). Even in gnome 45 was I face the same issue. Gnome 46 seemed to work so I went with it